onnxruntime/core/mlas/lib/qgemm.cpp

```diff
@@ -2655,11 +2661,66 @@ const MLAS_GEMM_U8X8_DISPATCH MlasGemmU8X8DispatchDefault = {
     nullptr,
     MLAS_GEMM_U8X8_KERNEL_DEFAULT::PackedK,
     0,
+    MLAS_GEMM_U8X8_KERNEL_DEFAULT::PackedStrides.M,
+    MLAS_GEMM_U8X8_KERNEL_DEFAULT::PackedStrides.N
 };
 
+/**
```

Comment:
can you update the style to follow what was already in the file? 4 space indent, parameter per line, commenting style

Reply:
> can you update the style to follow what was already in the file? 4 space indent, parameter per line, commenting style

I've tried to keep the 4 space indent, will take care of the one parameter per line. About this comment style for the function though, the nice thing about this new style is that in visual studio and vscode, when you hoover your mouse over the function name, this comments comes up in a tooltip. Would you consider it?